src/components/pages/divisionsPage/divisionsPage.tsx

```diff
@@ -0,0 +1,53 @@
+"use client";
+import {
+  Box,
+  Heading,
+  HorizontalRule,
+} from "@nypl/design-system-react-components";
+import PageLayout from "../../pageLayout/pageLayout";
+import { headerBreakpoints } from "src/utils/breakpoints";
+import SwimLanes from "src/components/swimlanes/swimLanes";
+import { props } from "__tests__/data/swimlaneProps";
+import useBreakpoints from "src/hooks/useBreakpoints";
+
+export default function DivisionsPage() {
+  const { isLargerThanLargeTablet, isLargerThanLargeMobile } = useBreakpoints();
+  const numColumns = isLargerThanLargeTablet
+    ? 4
+    : isLargerThanLargeMobile
+    ? 2
+    : 1;
+  return (
+    <PageLayout
+      activePage="divisions"
+      breadcrumbs={[
+        { text: "Home", url: "/" },
+        { text: "Divisions", url: "/divisions" },
+      ]}
+    >
+      <Box
+        sx={{
+          [`@media screen and (min-width: ${headerBreakpoints.smTablet})`]: {
+            maxWidth: "715px",
```

Comment:
Is this the value from figma? Seems a bit random.

Reply:
Yup